jumpscale/tools/nginx/nginxserver.py

```diff
@@ -64,3 +67,24 @@ def restart(self):
         """
         self.stop()
         self.start()
+
+    def get_pid_file_path(self):
+        """Return the path to nginx.pid file as specified in the Nginx configuration file
+
+        Raises:
+            j.exceptions.Runtime: if pid directive not found in the Nginx configuration file
+
+        Returns:
+            str: the path to the file that process ID of the Nginx master process is written to.
+        """
+        with open(j.sals.fs.expanduser(self.config_path), "r") as file:
+            for line in file:
+                m = re.match(r"^pid\s+(.*);", line)
+                if m:
+                    pid_file_path = m.group(1)
+                    break
```

Comment:
should be an immediate return here instead

Reply:
Done, thanks